kuma/urls.py

```diff
@@ -22,20 +22,41 @@
         lambda x: redirect('https://mozilla.org/contribute/events'),
         name='events'),
 
-    # Django admin:
-    url(r'^admin/wiki/document/purge/',
-        purge_view,
-        name='wiki.admin_bulk_purge'),
-    url(r'^admin/', include(admin.site.urls)),
+]
+
+if settings.MAINTENANCE_MODE:
+    urlpatterns.append(
+        url(r'^admin/.*', lambda x: redirect('maintenance_mode'))
```

Comment:
👍 for redirecting the admin, but I'd rather use ``RedirectView`` than a ``lambda``, such as http://stackoverflow.com/questions/15706489/redirect-to-named-url-pattern-directly-from-urls-py-in-django